### ISSUE TITLE
Non prise en compte dans la gestion des groupes des identifiants commençant par une @

### ISSUE THREAD

**Issue by jeey** [YesWiki/yeswiki#1108]
**Type of issue (keep only one) / Type de demande (ne garder qu'une ligne)**
Bug / Bogue

**Description**
Lorsque des droits de groupe sont utilisés pour des accès à certaines pages, ces droits ne sont pas pris en compte lorsque l'identifiant de l'utilisateur commence par une @ (ou contient une @ ?)

**Additionnal informations / Informations complémentaires**
- version of YesWiki / version de YesWiki : 4.4.2

Testé sur deux YesWiki (utilisant tous les deux une connexion via CAS) - non testé sur un Yeswiki avec le système d'identification habituel

**Comment by mrflos** [comment]
oui, le @ est réservé pour les groupes, je ne suis pas sur que ce soit trivial de supporter les @ pour les noms de users.. 
Je laisse l'issue pour regarder un jour, mais j'aurais pas le temps de suite, en attendant vaut mieux enlever le @ du nom du user sso

**Comment by jeey** [comment]
> en attendant vaut mieux enlever le @ du nom du user sso
C'est ce que j'ai fait en solution de contournement.

En todo, peut-être faudrait-il mettre un message d'invalidation du nom d'utilisateurice à la création ?
Mais en cas d'utilisation d'un CAS, ça ne résoudra pas le problème. Peut-être alors échapper le premier caractère si c'est une @ ? (mais quels effets de bord ... ?)
